e2e/nx-aws-cache-e2e/tests/aws-cache.encrypted.test.ts

```diff
@@ -0,0 +1,93 @@
+import { AwsCache } from '../../../packages/nx-aws-cache/src/tasks-runner/aws-cache';
+import { MessageReporter } from '../../../packages/nx-aws-cache/src/tasks-runner/message-reporter';
+import { Logger } from '../../../packages/nx-aws-cache/src/tasks-runner/logger';
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { createWriteStream } from 'fs';
+import { Decrypt, EncryptConfig } from '../../../packages/nx-aws-cache/src/tasks-runner/encryptor';
+import { pipeline, Readable } from 'stream';
+import { promisify } from 'util';
+
+describe('Test aws put and get unencrypted file', () => {
```

Comment:
is this e2e test or unit/integration (jest)? why isn't it located besides the code?

Reply:
Changed to integration tests with source code